khal/khalendar/vdir.py

```diff
@@ -0,0 +1,336 @@
+'''
+Based off https://github.com/pimutils/python-vdir, which is itself based off
+vdirsyncer.
```

Comment:
I guess that is totally up to you, if you want to be listed as the author, go ahead and fix that.